src/compose/SendButton.js

```diff
@@ -30,14 +30,31 @@ export default class SendButton extends React.Component {
 
   render() {
     const { disabled, onPress } = this.props;
-    const opacity = { opacity: disabled ? 0.25 : 1 };
+    const opacity = { opacity: 0.25 };
 
-    return (
-      <Touchable style={styles.wrapper} onPress={disabled ? undefined : onPress}>
-        <View style={[styles.button, opacity]}>
-          <IconSend size={16} color="white" />
+    /*
+      Since ReactNative's communication with the native mobile component is through a bridge that is
```

Comment:
Good explanation. Try writing one-line comments though (always possible with lots of editing) ;)